server/controllers/groups.js

```diff
@@ -87,6 +87,30 @@ const getUserData = (groupId, tiers) => {
 
 export const getUsers = (req, res, next) => {
   let promise = getUserData(req.group.id, req.group.tiers);
+
+  if (req.query.filter && req.query.filter === 'active') {
+    const now = moment();
+    promise = promise.filter(backer => now.diff(moment(backer.lastDonation), 'days') <= 90);
+  }
+
+  const isAdmin = !!(req.remoteUser && _.intersection(req.remoteUser.rolesByGroupId[req.group.id], ['HOST', 'MEMBER']).length > 0);
```

Comment:
We are introducing another term. Let's stick to the ones we have. `isHostOrMember` is better.